src/main/java/io/coala/jetbrains/analysis/CodeAnalysisRunner.java

```diff
@@ -149,4 +155,104 @@ public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType
+        codeAnalysisConsoleView.print(debugPrefix, prefixSeverity);
+      } else if (prefixSeverity == CodeInspectionSeverity.WARNING) {
+        codeAnalysisConsoleView.print(warningPrefix, prefixSeverity);
+      } else {
+        codeAnalysisConsoleView.print("\n", CodeInspectionSeverity.INFO);
+      }
+
+      codeAnalysisConsoleView.print(textToPrint, suffixSeverity);
+    } else {
+      if (prefixSeverity == CodeInspectionSeverity.WARNING) {
+        codeAnalysisConsoleView.print("\b\b\b\b\b\b\b\b\b" + warningPrefix, prefixSeverity);
+      } else if (prefixSeverity == CodeInspectionSeverity.ERROR) {
+        codeAnalysisConsoleView.print("\b\b\b\b\b\b\b" + errorPrefix, prefixSeverity);
+      } else if (prefixSeverity == CodeInspectionSeverity.DEBUG) {
+        codeAnalysisConsoleView.print("\b\b\b\b\b\b\b" + debugPrefix, prefixSeverity);
+      } else {
+        codeAnalysisConsoleView.print("\n", CodeInspectionSeverity.INFO);
+      }
+
+      codeAnalysisConsoleView.print(text, suffixSeverity);
+    }
+  }
+
+  private void addConsolePrinterListener(@NotNull OSProcessHandler processHandler) {
+    processHandler.addProcessListener(new ProcessAdapter() {
+      String previousOutput = "";
+
+      @Override
+      public void onTextAvailable(@NotNull ProcessEvent event, @NotNull Key outputType) {
+        final String text = event.getText();
+
+        if (text == null) {
+          return;
+        }
+
+        if (outputType.equals(ProcessOutputTypes.STDERR)) {
+          final String severityCheckText =
+              text.length() >= 10 ? text.substring(0, 10).toLowerCase() : null;
+          final int previousOutputLength = previousOutput.length();
+          final String previousOutputSeverityCheckText =
+              previousOutput.length() >= 7 ? previousOutput.substring(previousOutputLength - 7)
+                  .toLowerCase() : "";
+
+          if (severityCheckText != null) {
+            if (severityCheckText.contains("[error]")) {
+              printToConsole(text, CodeInspectionSeverity.ERROR,
+                  CodeInspectionSeverity.INFO, false);
+            } else if (severityCheckText.contains("[debug]")) {
+              printToConsole(text, CodeInspectionSeverity.DEBUG,
+                  CodeInspectionSeverity.INFO, false);
+            } else if (severityCheckText.contains("[warning]")) {
+              printToConsole(text, CodeInspectionSeverity.WARNING,
+                  CodeInspectionSeverity.INFO, false);
+            } else if (previousOutputSeverityCheckText.contains("[error]")) {
+              printToConsole(text, CodeInspectionSeverity.ERROR,
+                  CodeInspectionSeverity.INFO, true);
+            } else if (previousOutputSeverityCheckText.contains("[debug]")) {
+              printToConsole(text, CodeInspectionSeverity.DEBUG,
+                  CodeInspectionSeverity.INFO, true);
+            } else if (previousOutputSeverityCheckText.contains("arning]")) {
```

Comment:
`[warning]`?

Reply:
Just checking the last 7 characters, that's why I said definitely not elegant.